### PR TITLE
CMS ARS 3.1 AWS RDS Moderate Overlay

### PR DESCRIPTION
**Pull request by karikarshivani**
Signed-off-by: karikarshivani <karikarshivani@gmail.com>

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+copyright: MITRE SAF
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: "An InSpec Overlay Profile for CMS ARS 3.1 Moderate to validate the Microsoft SQL Server 2017 CIS Benchmark"
+version: 1.2.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: microsoft-sql-server-2017-cis-baseline
+  url: https://github.com/mitre/microsoft-sql-server-2017-cis-baseline/archive/v1.2.0.tar.gz
```
Comment (karikarshivani) [inline inspec.yml]: I'll update the branch name in the `depends.url` when the PR for the baseline has been approved and merged. Thanks!

**Review by ejaronne — CHANGES_REQUESTED** [review]
update depends reference to main.tar.gz (not 1.2.0)
update README exec instructions to use main.tar.gz (not master)
update README "full heimdall server" url to point to heimdall2 repo not old heimdall

**Review by ejaronne — APPROVED** [review]
typo corrections in README and inspec.yml added

File: inspec.yml (hunk 1 above)
Comment (karikarshivani) [reply inspec.yml]: Branch name updated

**Comment by karikarshivani** [comment]
Profile tested - ready to be merged. Thanks!

**Pull request merged**